packages/shared/lib/core/app/actions/downloadAppUpdate.ts

```diff
@@ -123,5 +98,33 @@ export function downloadAppUpdate(): void {
         }
     })
 
+    const cleanup = () => {
```

Comment:
Let's make this a regular function as well

Reply:
Yes, I think this makes sense. The function is used in the callback for the unsubscribers themselves, which seems to be / work okay although it makes it a bit weird to extract it out.